app/src/main/java/com/gsandchase/mileagelogger/Logger.java

```diff
@@ -1,81 +0,0 @@
-package com.gsandchase.mileagelogger;
```

Comment:
This class seems unnecessary - Log.d()/Log.v() etc. can be used for debugging purposes. They write to a systemwide logfile that can be examined with logcat.